.github/workflows/actions.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: "0 0 1 * *"
```

Comment:
To check if things build, I'd suggest to only build on "pull_request" and on "workflow_dispatch" (the latter allows triggering manually on any branch). 
We maybe could keep "push" but I don't think that's necessary for this repo - with "workflow_dispatch" the run can be triggered manually when desired.
I also don't think we really need the scheduled builds... or are you thinking of this as tracking possible issues with  GHA setup? Still, I personally don't think that's really needed.